model.py

```diff
@@ -2205,18 +2205,15 @@ def add_link(self, rel, href, data_source, media_type=None, content=None,
         if content or content_path:
             # We have content for this resource.
             resource.set_fetched_content(media_type, content, content_path)
-        elif (media_type and (
-                not resource.representation
-                or not resource.representation.mirrored_at)
-        ):
-            # There's a version of this resource stored elsewhere that we
-            # can use.
-            #
-            # TODO: just because we know the type and URL of the
-            # resource doesn't mean we can actually serve that URL
-            # to patrons. This needs some work.
-            resource.set_mirrored_elsewhere(media_type)
```

Comment:
Important change -- we no longer act like a resource we just heard of is 'mirrored' at its original URL.